src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -40,8 +45,17 @@ object Boot extends App with LazyLogging {
     implicit val materializer = ActorMaterializer()
     import scala.concurrent.ExecutionContext.Implicits.global
 
-    val accessPolicyDAO = new JndiAccessPolicyDAO(directoryConfig)
-    val directoryDAO = new JndiDirectoryDAO(directoryConfig)
+    val dirURI = new URI(directoryConfig.directoryUrl)
```

Comment:
jndi supports a ldap url which is what we have in conifg. Unboundid does not so here is a little tap dance to figure it out.